### PR TITLE
Add Issue bug template

### PR DESCRIPTION
**Pull request by clemens-tolboom**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug", "triage"]
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Seems unnecessary to me. I think the title should already shortly describe this, rest in how to reproduce

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug", "triage"]
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Don't think you automatically want to label things as bug.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: I'd leave out the example. You have to delete this every time. Will become annoying. Maybe we could add an example of how to report an issue to the `CONTRIBUTING.md`. That might be nice.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Missing the `** Observed behaviour**`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: We prefer "behaviour"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: I'd skip this. If relevant, we can ask.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (marikaris) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: I'd skip this. Screenshots can be added in the how to reproduce/observed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (clemens-tolboom) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I do ... hence the triage so we have to check it first ... easy to find unreviewed bugs

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (clemens-tolboom) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: I do not think people read the fine manuals first.

I agree with 'always delete' but seen many projects too ... let's see what happens?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (clemens-tolboom) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: I think this help reporting as a contemplating part of reporting a bug

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 6 above)
Comment (clemens-tolboom) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: I'd like to keep this in as when reading a bug report it is already there so we can continu resolving it.

**Review by marikaris — APPROVED** [review]

**Pull request merged**